### ISSUE TITLE
20.08: install.sh fails, bin/jjs does not exist

### ISSUE THREAD

**Issue by tinywrkb** [flathub/org.freedesktop.Sdk.Extension.openjdk#5]
`org.freedesktop.Sdk.Extension.openjdk//20.08`, running the usual `/usr/lib/sdk/openjdk/install.sh`, it fails with the error

```
cp: cannot stat 'bin/jjs': No such file or directory
Error: module openjdk: Child process exited with code 1
```

and indeed, `bin/jjs` is missing from the extension.

**Comment by mbooth101** [comment]
Thanks for the report, I'll take a look now

**Comment by mbooth101** [comment]
Should be fixed in 652cfb9fd62ee1f80ffc8998a7d95848627d6e5b
